src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,157 @@
+@import "../styles/components/_variables";
+
+::selection {
+    background: $yellow; /* WebKit/Blink Browsers */
+}
+::-moz-selection {
+    background: $yellow; /* Gecko Browsers */
+}
+h3 {
+    text-align: center;
+    font-weight: $font-bold;
+    color: $blue;
+}
+h2 {
+    font-weight: $font-bold;
+}
+
+.mosquito {
+    width: 80px;
+    height: auto;
+}
+
+.lead-title {
+    font-weight: $font-bold;
+    font-size: 2.5em;
+    color: $black;
+}
+
+.lead-sub-heading {
+    color: grey;
+    font-weight: $font-bold;
+    font-size: 1.5em;
+}
+
+.rectangle-speech-border {
+    position: relative;
+    padding: 50px;
+    width: 100%;
+    margin: 1em 0 3em;
+    border: 2px solid $black;
+    color: #333;
+    background: $white;
+    /* css3 */
+    -webkit-border-radius: 50px;
+    -moz-border-radius: 50px;
+    border-radius: 50px;
+}
+
+/* creates larger curve */
+.rectangle-speech-border:before {
```

Comment:
these pseudo-elements should be included within the `.rectangle-speech-border` block (sass nesting)

also, the `before` & `after` shares lots style in common, can also consider
```
&:before,
&:after {
// common styles
}

&:before {
// unique to before
}

&:after {
// unique to after
}
```